sdc/datatypes/hpat_pandas_series_functions.py

```diff
@@ -3322,6 +3322,10 @@ def hpat_pandas_series_quantile(self, q=0.5, interpolation='linear'):
 
     Pandas API: pandas.Series.quantile
 
+    Limitations
+    -----------
+    Parameter interpolation is currently unsupported by Intel Scalable Dataframe Compiler
```

Comment:
```suggestion
    Parameter ``interpolation`` is currently unsupported.
```

Reply:
Actually this parameter is supported, but only with default value.